### PR TITLE
Prisma Cloud Compute Solution Package With CCP CLV2

### PR DESCRIPTION
**Pull request by v-amolpatil**
Required items, please complete
   
   Change(s):
   - Added files and package for PrismaCloudCompute with files and CCP connector support added manuall.

   Reason for Change(s):
   - Need CCP CLV2 package for this solution.

   Version Updated:
   - 3.0.0

   Testing Completed:
   - Yes
   
   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

After custom deployment of Maintemplate:
image

image

### CODE REVIEW

**Review by v-sabiraj — DISMISSED** [review]

File: Solutions/PrismaCloudCompute/Data Connectors/dataConnector1.json
Comment (Tichandr) [inline Solutions/PrismaCloudCompute/Data Connectors/dataConnector1.json:1]: Can the name be better?

Comment (v-amolpatil) [reply Solutions/PrismaCloudCompute/Data Connectors/dataConnector1.json:1]: renamed

File: Solutions/PrismaCloudCompute/Data Connectors/dataConnectorPoller.json
Comment (Tichandr) [inline Solutions/PrismaCloudCompute/Data Connectors/dataConnectorPoller.json:1]: Can this file be array of objects?

File: Solutions/PrismaCloudCompute/Data Connectors/connectorDefinition.json [hunk 1]
```diff
@@ -0,0 +1,117 @@
+{
+    "name": "PrismaCloudComputeDefinition",
+    "apiVersion": "2022-09-01-preview",
```
Comment (Tichandr) [inline Solutions/PrismaCloudCompute/Data Connectors/connectorDefinition.json:3]: id property is mandate. check for api supportability of the key in clv2

File: Solutions/PrismaCloudCompute/Data Connectors/dataConnectorPoller.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+    "type": "Microsoft.SecurityInsights/dataConnectors",
+    "apiVersion": "2022-10-01-preview",
+    "name": "apiRequest",
+    "kind": "RestApiPoller",
+    "properties": {
+        "connectorDefinitionName": "connectorDefinitionName",
```
Comment (Tichandr) [inline Solutions/PrismaCloudCompute/Data Connectors/dataConnectorPoller.json:7]: The value should be the connectordefinition id here

File: Solutions/PrismaCloudCompute/Data Connectors/DCR.json
Comment (Tichandr) [inline Solutions/PrismaCloudCompute/Data Connectors/DCR.json:1]: For multiple DCR's, will be kept under single file as array of objects. so keep this file as standard array type. Same thing goes for poller file and table

File: Solutions/PrismaCloudCompute/Data Connectors/dataConnectorPoller.json (hunk 2 above)
Comment (v-amolpatil) [reply Solutions/PrismaCloudCompute/Data Connectors/dataConnectorPoller.json]: done

File: Solutions/PrismaCloudCompute/Data Connectors/connectorDefinition.json (hunk 1 above)
Comment (v-amolpatil) [reply Solutions/PrismaCloudCompute/Data Connectors/connectorDefinition.json:3]: done

File: Solutions/PrismaCloudCompute/Data Connectors/dataConnectorPoller.json
Comment (v-amolpatil) [reply Solutions/PrismaCloudCompute/Data Connectors/dataConnectorPoller.json:1]: done

File: Solutions/PrismaCloudCompute/Data Connectors/DCR.json
Comment (v-amolpatil) [reply Solutions/PrismaCloudCompute/Data Connectors/DCR.json:1]: As we are not able to do array shown in below screenshot in brown color so making array from start of the file highlighted in orange color. Please let me know if this is what expected:
image

Comment (v-amolpatil) [reply Solutions/PrismaCloudCompute/Data Connectors/DCR.json:1]: done

**Review by daxesht — APPROVED** [review]

**Pull request merged**